Show imports in custom theme examples

codesoap opened the pull request:
I was briefly confused about the "color" package, since I couldn't find it within fyne. I think showing the imports could save some confusion.

andydotxyz commented:
Hmm, interesting. I agree it would be good to avoid confusion - but by adding the imports the code snippets look like full files which they are not (no package etc).
Perhaps it could be mentioned in the text instead?

codesoap commented:
Fair point.

If I'm honest, I must admit that I didn't even read most of the text, I mostly just looked at the code snippets when trying to understand how custom themes work. Thus, I have now changed the suggestion to mention the `color` package in a comment. What do you think about this approach?

Most readers probably already know where `fyne` and `theme` come from, so I removed mention of that to not bloat the documentation.

andydotxyz reviewed (approved):
Good approach thanks
